packages/devtools_app/lib/src/screens/app_size/app_size_controller.dart

```diff
@@ -51,6 +51,9 @@ class AppSizeController {
   ValueListenable<TreemapNode?> get analysisRoot => _analysisRoot;
   final _analysisRoot = ValueNotifier<TreemapNode?>(null);
 
+  ValueListenable<TreemapNode?> get originalRoot => _originalRoot;
+  final _originalRoot = ValueNotifier<TreemapNode?>(null);
```

Comment:
Let's call this `appRoot`

Reply:
We shouldn't need to track another notifier / listenable pair for this. The root should be accessible from the TreemapNode (see TreeNode.root). Take a look at the BreadcrumbNavigator we use in the app size page. The breadcrumb navigator is able to calculate the path from the current node to the root, so this should be possible without tracking another var.